packages/x-components/static-docs/experience-search-and-discovery/web-local-storage.md

```diff
@@ -6,10 +6,9 @@ tags:
   - cookies
   - local storage
   - history queries
+  - cookieless
```

Comment:
Maybe we should add the `privacy` tag here too.

Reply:
sounds good! I need to revise content in the coming days, I can add it then :)